mixed/spark/common/pom.xml

```diff
@@ -215,43 +215,16 @@
 
     </dependencies>
 
+    <build>
```

Comment:
I haven't seen the new Scala code. Why do we need this plugin?

Reply:
fixed.